bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -15,6 +15,8 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{{_('Welcome to Firefox Developer Edition')}}{% endblock %}
 {% block page_title_suffix %}{% endblock %}
+{% block page_image %}{{ static('img/firefox/developer/page-image.png') }}{% endblock %}
```

Comment:
There's no harm in adding this image here, but `/firstrun` pages should not form part of a social campaign, since they are treated as in-product pages only accessible to Firefox. Probably good to make sure folks understand this when asking for changes.